lib/model.js

```diff
@@ -959,6 +959,10 @@ Model.prototype.sync = function(options) {
     , attributes = this.tableAttributes;
 
   return Promise.try(function () {
+    if (options.hooks) {
```

Comment:
This is the `model.sync` call right?
You do a condition based on `options.hooks` but you have not set a default value for it as far as i can tell so it will effectively be false where most other calls have `true` as a default.